sorc/machine-setup.sh

```diff
@@ -98,7 +98,15 @@ elif [[ -d /data/prod ]] ; then
     fi
     target=s4
     module purge
+elif [[ -z ${PW_CSP} ]]; then
+    if [[ "${PW_CSP}" == "aws" ]]; then
+        target=noaacloud
```

Comment:
You set 'target', but it does not appear to be used downstream.

Reply:
I added downstream logic to use this bit now.